src/components/Bridge/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { CHAINS, ChainId, Token } from '@pangolindex/sdk';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Tab, TabList, TabPanel, Tabs } from '../Tabs';
+import BridgeCard from './BridgeCard';
+import BridgeRoute from './BridgeRoute';
+import { Step } from './BridgeRoute/types';
+import BridgeTransfer from './BridgeTransfer';
+import { BridgeState } from './BridgeTransfer/types';
+import { PageWrapper, Routes, Transactions, Transfers } from './styles';
+
+export enum BridgePrioritizations {
+  recommended,
+  fast,
+  normal,
+}
+
+const Bridge = () => {
+  const [tabIndex, setTabIndex] = useState(0);
+  const steps: Step[] = [
+    {
+      contractType: 'LI.FI Contract',
+      subSteps: ['1. Swap to 0.0538 USDT via PANGOLIN', '2. Transfer to 0.0522 USDT via PANGOLIN'],
+    },
+    {
+      contractType: 'LI.FI Contract',
+      subSteps: ['1. Swap to 0.0538 USDT via DODO'],
+    },
+  ];
+
+  const { t } = useTranslation();
+  const currency0 = new Token(
+    ChainId.AVALANCHE,
+    CHAINS[ChainId.AVALANCHE].contracts!.png,
+    18,
+    CHAINS[ChainId.AVALANCHE].png_symbol!,
+    'Pangolin',
+  );
+  const currency1 = new Token(
+    ChainId.AVALANCHE,
+    CHAINS[ChainId.AVALANCHE].contracts!.png,
+    18,
+    CHAINS[ChainId.AVALANCHE].png_symbol!,
+    'Pangolin',
+  );
+
+  return (
+    <PageWrapper>
+      <BridgeCard />
+      <Transactions>
+        <Tabs
```

Comment:
Text is not visible in Tab

![Screenshot 2022-09-28 at 5 38 07 PM](https://user-images.githubusercontent.com/6604146/192775059-4358403d-e30f-4c27-b3d0-56063220495c.png)

Reply:
Done